app/actions/service_credential_binding_app_create.rb

```diff
@@ -32,7 +32,8 @@ def precursor(service_instance, message:, app: nil, volume_mount_services_enable
 
         ServiceBinding.new.tap do |b|
           ServiceBinding.db.transaction do
-            binding.destroy if binding
+            binding&.destroy
```

Comment:
Probably we could use a normal if block here instead of using `binding&` and `if binding`?